<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>850</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GUSS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>750</width>
      <height>350</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="env">
     <attribute name="title">
      <string>Environment Settings</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>6</y>
        <width>721</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="apiURLlabel">
         <property name="text">
          <string>API URL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="env_api_base_url">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>https://broadbandmap.fcc.gov</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>username</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="env_username"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aPIKEYLabel">
         <property name="text">
          <string>API KEY</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="env_api_key"/>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="env_file_btn">
         <property name="text">
          <string>Load Env File</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="base_file_path">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>631</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="base_folder_select_btn">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>230</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Base Output directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Mobile">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Mobile</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>291</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="m_asOfDate_lbl">
         <property name="text">
          <string>As of Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="m_asOfDate_com">
         <item>
          <property name="text">
           <string>2021-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-12-31</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Provider Id List</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="m_providerIDList">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>State Fips List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="m_state_fips_list">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Technology List</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="m_tech_list">
         <property name="text">
          <string>300, 400, 500</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Technology Type</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="m_tech_type_com">
         <item>
          <property name="text">
           <string>Mobile Broadband</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mobile Voice</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>GIS Coverage Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="m_GisCoverageType_com">
         <item>
          <property name="text">
           <string>Hexagon Coverage</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Raw Coverage</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>5G Speed Teir List</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="m_5G_speed_list">
         <property name="text">
          <string>7/1, 35/3</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>GIS File Type</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="m_GIS_Output_type_com">
         <item>
          <property name="text">
           <string>shp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gpkg</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="m_submitt">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Download Mobile coverages</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Fixed">
     <attribute name="title">
      <string>Fixed</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>224</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="f_asOfDate_lbl">
         <property name="text">
          <string>As of Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="f_asOfDate_com">
         <item>
          <property name="text">
           <string>2021-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-12-31</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Provider Id List</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="f_providerIDList">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>State Fips List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="f_state_fips_list">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Technology List</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="f_tech_list">
         <property name="text">
          <string>0, 10, 40, 50, 60, 61, 70, 71, 72</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="f_polygonize_radio">
         <property name="text">
          <string>Create Fixed Coverage Polygons</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="f_submitt">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Download Files</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="gISFileTypeLabel">
         <property name="text">
          <string>GIS File Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="f_GIS_Output_type_com">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>shp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>gpkg</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Challenge">
     <attribute name="title">
      <string>Challenge</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>133</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="c_asOfDate_lbl">
         <property name="text">
          <string>As of Date</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="c_asOfDate_com">
         <item>
          <property name="text">
           <string>2021-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2022-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2023-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2024-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2025-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2026-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2027-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2028-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2029-12-31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-06-30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2030-12-31</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="c_category_com">
         <item>
          <property name="text">
           <string>Fabric Challenge - In Progress</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fabric Challenge - Resolved</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fixed Challenge - Cumulative</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fixed Challenge - In Progress</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fixed Challenge - Resolved</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mobile Challenge - In Progress</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mobile Challenge - Resolved</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>State Fips List</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="c_state_fips_list">
         <property name="text">
          <string>all</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="c_submitt">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Download Files</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="header">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="HeaderName">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Guss: The FCC BDC API</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="app_stdOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>881</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>770</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_messages_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Logs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <property name="toolTipsVisible">
     <bool>true</bool>
    </property>
    <addaction name="actionLoad_Enviroment_file"/>
    <addaction name="separator"/>
    <addaction name="actionRestart_App"/>
    <addaction name="actionQuit_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionload_env">
   <property name="text">
    <string>load Environment</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Enviroment_file">
   <property name="text">
    <string>Load Enviroment file</string>
   </property>
  </action>
  <action name="actionRestart_App">
   <property name="text">
    <string>Restart App</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSearch_Logs">
   <property name="text">
    <string>Search Logs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
